2주차/11866/11866_python_제로.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+n, k = map(int, sys.stdin.readline().split())
+lst = list(range(1, n + 1))
+visited = [False] * (n + 1)
+current = k - 1
+count = 0
+answer = [lst[current]]
+visited[current] = True
+
+while True:
+    if len(answer) == n:
+        break
+    current += 1
+    if current >= n:
+        current = n - current
+    if visited[current] == False:
```

Comment:
```suggestion
    if not visited[current]:
```
궁금하면 파이썬 pep-8 스타일 가이드의 Programming Recommendations 단락을 살펴보면 좋을 것 같네요.
https://peps.python.org/pep-0008/